### ISSUE TITLE
Json for more models

### ISSUE THREAD

**Issue by rasmusjagren** [chrisc123/jebao_aqua-homeassistant#6]
Hi! Thanks a lot for taking this on. I wonder where I can find the info in the json file for or models. I manually changed the product id to match mine, but the offsets don’t match (I think). Thanks!

Btw, there can’t be that many models, would be great if you could put all the files there:)

Cheers,
Rasmus

**Comment by VMartovsky** [comment]
Yes, I would really like to know where the json file comes from and its structure? This would speed up adding new devices.

**Comment by chrisc123** [comment]
Hi, so it's a modified version of the JSON that's returned from the Gizwits API endpoint - e.g.:
curl -X GET "https://euapi.gizwits.com/app/datapoint?product_key=f0d844ab0d4947ac9527a286160bc705"

However - this contains a bunch of stuff that isn't supported by the integration as yet (e.g. Timer/Schedule functionality) and is also in Chinese. So at the moment I just manually review/edit that JSON, translate and convert Chinese fields and restructure the 'desc' elements into arrays so that HA has some English text to use for multivariate values.

Note - it's important to keep the 'enum' elements in that JSON unmodified and in Chinese as those are the values actually sent to Gizwits API for control!

The problem is there's no obvious (to me!) way to map the product keys to Jebao models - the descriptions/names of the pumps don't really match so although it's possible to enumerate all the product keys from the Android APK, I don't really know which correspond to MLW/SLW/EP etc without someone actually having one of those devices and providing the logs.

**Comment by VMartovsky** [comment]
Thank you for the detailed explanation! Now everything is clearer to me!
